packages/components/src/components/date-picker/date-picker.css

```diff
@@ -100,6 +102,7 @@ duet-date-picker .duet-date__toggle:active scale-icon-content-calendar {
 }
 
 .scale-date-picker {
+  --z-index-label: 10;
```

Comment:
should we add this to the top of the file, or is there a reason you've inserted a new variable into this position? thanks :)

Reply:
@marvinLaubenstein I agree with Daniel this line should be moved up, to place it inside `:host`, around line 34. Other than that, this looks fine!